Gems/Atom/Feature/Common/Assets/Shaders/Materials/BasePBR/BasePBR_LightingData.azsli

```diff
@@ -68,38 +63,18 @@ class LightingData_BasePBR
     real diffuseAmbientOcclusion;
     real specularOcclusion;
 
-    void Init(float3 positionWS, real3 specularNormal, real roughnessLinear, float3 viewPosition[MAX_SHADING_VIEWS]);
+    // Note: The intent is that the LightingData_BasePBR definition is included in BasePBR_Defines.azsli, then the render pipeline 
+    // declares the SampleBrdfMap - function in the PipelineCallbacks.azsli, and then we use it in LightingData_BasePBR::Init(), which *should* be placed
+    // in BasePBR_LightingEval.azsli (which is included via BasePBR.azsli).
+    // Unfortunately AZSLC messes up with class member functions and either drops the SampleBRDFMap - function, or crashes, which forces
+    // us to move the BRDF-Initialisation to the non-class function BasePBR_InitLightingData().
+    // void Init(float3 positionWS, real3 specularNormal, float3 viewPosition[MAX_SHADING_VIEWS]);
```

Comment:
Is the plan to fix this in the future? Is that why we are leaving it commented out here.